### PR TITLE
Fix fast quality change for alternate audio

### PR DESCRIPTION
**Pull request by gesinger**

### CODE REVIEW

File: src/master-playlist-controller.js [hunk 1]
```diff
@@ -698,7 +698,7 @@ export class MasterPlaylistController extends videojs.EventTarget {
       this.masterPlaylistLoader_.media(media);
 
       this.mainSegmentLoader_.resetLoader();
-      if (this.audiosegmentloader_) {
```
Comment (mjneil) [inline src/master-playlist-controller.js]: not only was this not checking for the playlist loader, it's not cased properly for the segment loader lol

File: src/master-playlist-controller.js [hunk 2]
```diff
@@ -698,7 +698,7 @@ export class MasterPlaylistController extends videojs.EventTarget {
...
+      if (this.audioPlaylistLoader_) {
```
Comment (mjneil) [inline src/master-playlist-controller.js]: Why should this be `audioPlaylistLoader_` vs `audioSegmentLoader_`?

Comment (gesinger) [inline src/master-playlist-controller.js]: audioSegmentLoader_ always exists, even if not being used. audioPlaylistLoader_ only exists if we are using alt audio

**Comment by mjneil** [comment]
Is it even necessary to reset the audio loader on fast quality change? If `fastQualityChange_` causes a rendition switch, the `mediachange` event listener will call `setUpAudio` if the active audio group has changed. `setUpAudio` will tear down the playlist loader and reset the segment loader

**Comment by imbcmdth** [comment]
We decided to remove the code if it is redundant and to add a test to prove that it was not necessary.

**Pull request merged**